tests/checks/mock/test_spark.py

```diff
@@ -93,7 +93,8 @@ class SparkCheck(AgentCheckTest):
 
     SPARK_CONFIG = {
         'resourcemanager_uri': 'http://localhost:8088',
-        'cluster_name': CLUSTER_NAME
+        'cluster_name': CLUSTER_NAME,
+        'spark_cluster_mode': 'spark_yarn_mode'
```

Comment:
A standalone mode test would be fantastic at some point. It's not something that I believe should stop this PR from getting merged, but we should keep in mind.